Dockerfile

```diff
@@ -0,0 +1,13 @@
+#docker build -t damare .
+
+FROM node:lts-buster
+
+RUN apt update && apt install -y vim wget open-jtalk
+RUN git clone https://github.com/Chipsnet/damare.git
+WORKDIR /damare
+# RUN git checkout Chipsnet/issue3
```

Comment:
You should delete this line before merging this file to the master branch. This line is for testing.

Reply:
👀👍